util/test/checkChplInstall

```diff
@@ -25,16 +25,30 @@
 # To use something other than $HOME/.chpl, set the CHPL_CHECK_INSTALL_DIR
 # environment variable.
 
+function log_date()
+{
+    echo -n "$(date '+%Y-%m-%d %H:%M:%S') "
+}
 
-# Base name for test job in $CHPL_HOME/test/release/examples
-TEST_JOB_BASENAME=hello6-taskpar-dist
+function log_info()
+{
+    local msg=$@
+    log_date
+    echo "[INFO] ${msg}"
+}
 
-err() {
-    echo "error: $@" >&2
+function log_error()
+{
+    local msg=$@
+    log_date
+    echo "[ERROR] ${msg}" >&2
 }
 
+# Base name for test job in $CHPL_HOME/test/release/examples
+TEST_JOB_BASENAME=hello6-taskpar-dist
+
 # Notify user we are running this script
-echo "Running minimal test script: checkChplInstall"
+log_info "Running minimal test script: checkChplInstall"
```

Comment:
Also not new, but might be nice to put the full path to checkChplInstall. `$CHPL_HOME/util/test/checkChplInstall`